Packs/CortexXDR/ReleaseNotes/6_1_71.md

```diff
@@ -0,0 +1,27 @@
+
+#### Playbooks
+
+##### New: Cortex XDR disconnected endpoints
```

Comment:
Hi @ali-sawyer !
The playbook is not new. Please only add the changes made to the playbook.